docker-local/start-linksharing.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -xe
+
+cd /tmp
+wget --output-document=GeoLite2-City.mmdb \
+    https://link.storjshare.io/s/jxy4d5ux5wvc2jf7bmt2rmlnfeeq/oliosite/public/GeoLite2-City.mmdb?download
```

Comment:
This absolutely can not be commited to main and actually is illegal to do..
Please remove asap.

Reply:
Sorry I was not aware.

I will check if it works without.